core/pref.h

```diff
@@ -22,9 +22,9 @@ typedef struct
 } partial_pressure_graphs_t;
 
 typedef struct {
-	char *access_token;
-	char *user_id;
-	char *album_id;
+	const char *access_token;
```

Comment:
have you enabled `-Wall` for your toolchain to see if some of these changes introduce cast warnings?

Reply:
Tried `-Wall`. Every single file including "dive.h" gives a signed/unsigned comparison warning. But no new cast warnings.

Reply:
yeah, the signed / unsigned i (we) know of. i was hoping that we don't introduce more warnings.